Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/view/plan_create/PlanCreateFragment.java

```diff
@@ -35,20 +45,23 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         AppComponent appComponent = ((App) getActivity().getApplication()).getAppComponent();
         appComponent.inject(this);
 
-        PlanCreateData planData = new PlanCreateData("");
+        planData = new PlanCreateData("");
         binding.setPlanData(planData);
         binding.setPlanDataClick(this);
         return view;
     }
 
     private Long addPlan(String planName) {
-        try {
-            long planId = planTemplateRepository.create(planName);
-            showToastMessage(R.string.plan_saved_message);
-            return planId;
-        } catch (RuntimeException exception) {
-            return handleSavingError(exception);
+        if (validateName(planName)) {
```

Comment:
Nice job 👍  could you please add test for it?

Reply:
Sure - great idea :) thanks for the review! :)